_data/mentors.yml

```diff
@@ -2262,3 +2262,155 @@
     extra: Potential Mentoring Topics such as Software Development Strategies such as Best practices for planning, executing, and managing software projects effectively. .NET Development such as Deep dive into C#, ASP.NET MVC, and building robust applications using the .NET framework. ReactJS such as Developing dynamic and responsive web applications with ReactJS.
   network:
     - linkedin: https://www.linkedin.com/in/s-b-b51a76317/
+
+- name: Andrew  King
+  disabled: false
+  matched: false
+  sort: 200
+  num_mentee: 0
+  hours: 4
+  type: long-term
```

Comment:
@IdayatSanni I guess this is not updated, here is ad-hoc only and about his availability.